packages/block-library/src/group/block.json

```diff
@@ -19,6 +19,7 @@
 			"gradients": true,
 			"linkColor": true
 		},
-		"__experimentalPadding": true
+		"__experimentalPadding": true,
+		"customCSS": true
```

Comment:
Should I make it experimental to start with?

Reply:
Let's make it experimental. That way we have some more time to settle the matter of automatic / opt-in / opt-out solutions for front-end transformations.